src/templates/login.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Login Page</title>
+  </head>
+  <body>
+    <header class="login-header">
+      <nav>
+            <p class="homepage-link"><a href="./todo">DONE.</a></p>
+          <ul class="right-links">
+          <li>
+            <a href="./login">Log In</a>
+          </li>
+            <a href="./signup">Sign Up</a>
+          </li>
+          </ul>
+      </nav>
+    </header>
+    <form class="flex-center-login-form">
+      <h2>Log in to your account</h2>
+      <container>
```

Comment:
is `<container>` a thing? did you mean something like `<div class="form-container">`

Reply:
Ha! I was fishing around on the internet for alternatives to div and seem to have just invented this one.